extensions/mssql/src/objectManagement/ui/dropDatabaseDialog.ts

```diff
@@ -49,11 +49,11 @@ export class DropDatabaseDialog extends ObjectManagementDialogBase<Database, Dat
 	}
 
 	protected override async saveChanges(contextId: string, object: ObjectManagement.SqlObject): Promise<void> {
-		await this.objectManagementService.dropDatabase(this.options.connectionUri, this.options.objectUrn, this._dropConnections, this._deleteBackupHistory, false);
+		await this.objectManagementService.dropDatabase(this.options.connectionUri, this.options.database, this.options.objectUrn, this._dropConnections, this._deleteBackupHistory, false);
 	}
 
 	protected override async generateScript(): Promise<string> {
-		return await this.objectManagementService.dropDatabase(this.options.connectionUri, this.options.objectUrn, this._dropConnections, this._deleteBackupHistory, true);
+		return await this.objectManagementService.dropDatabase(this.options.connectionUri, this.options.database, this.options.objectUrn, this._dropConnections, this._deleteBackupHistory, true);
```

Comment:
Does the object URN not contain the DB name already?

Reply:
It does, but that creates another parsing step with its own edge cases. I've been thinking of removing objectURN in these calls since it's usually redundant

Reply:
There's lots of other places in the admin code that has a similar URN/Name redundancy

Reply:
SFC should have stuff to help with parsing, but if you're removing it I guess that's a moot point. It'd be nice to clean this up to only use one now though while you're looking at it. 

But not blocking, so I'll leave it up to you when that work gets done.

Reply:
I'd have to do another STS release to update that here, so I'll merge this as-is and clean up the URN stuff in a subsequent PR.